rest/servers/spring/src/main/java/com/connexta/store/rest/spring/StoreApi.java

```diff
@@ -95,8 +99,8 @@
           @NotNull
           @Pattern(regexp = SEMANTIC_VERSION_REGEX)
           String acceptVersion,
-      @Pattern(regexp = "^[0-9a-zA-Z]+$")
-          @Size(min = 32, max = 32)
+      @Pattern(regexp = UUID_REGEX)
+          @Size(min = 36, max = 36)
           @ApiParam(value = "The ID of the Dataset.", required = true)
           @PathVariable("datasetId")
           String datasetId,
```

Comment:
Can the type of this param be `UUID` instead of using the regex?

Reply:
Yup, seemed to work fine.